### ISSUE TITLE
Code Blocks in asciidoc

### ISSUE THREAD

**Issue by maximiliankolb** [theforeman/foreman-documentation#631]
Hi!
Running `git grep -h -B 1 "\-\-\-\-" guides | sort -u | grep "^\["` shows that we have quite a few different ways to format/configure code blocks. Is there a preferred way?

I tend towards `[options="nowrap", subs="verbatim,quotes,attributes"]` for readability, with the option to prefix a certain highlighting, for example `bash`, `yaml`, or `ruby`: `[ruby, options="nowrap", subs="verbatim,quotes,attributes"]`.

If we come to a conclusion, I would be happy to extend the [guides/README.md](https://github.com/theforeman/foreman-documentation/blob/master/guides/README.md#contribution-guidelines) and create a PR.

**Comment by melcorr** [comment]
Hey @maximiliankolb 

I think that they very much depend upon the actual code. I tend to leave out the verbatim, because **I think** that if you, for example, want to mark up a hostname, username, password etc is a variable value that we expect the user to change for themselves, instead of printing _username_ it will print verbatim and display the underscores. 

I did notice a funny quirk with our site where "quotes" highlights a string of text. I have fixed an instance of it here:  https://github.com/theforeman/foreman-documentation/pull/606/ - it was quotes in combination with the ####### block. 

I am quite happy, if I'm honest, to allow for any combination of `[options="nowrap", subs="verbatim,quotes,attributes"]` - for example, sometimes there are no attributes in the code block. 

As long as people test, I'm pretty sure we will be OK.
